package.json

```diff
@@ -10,13 +10,9 @@
     "prestart-production": "gulp build -p",
     "start-production": "node pm2Start",
     "lint": "npm run lint-js && npm run lint-json",
-    "lint-challenges": "jsonlint-cli seed/challenges/**/*.json",
-    "lint-server": "jsonlint-cli server/*.json",
-    "lint-resources": "jsonlint-cli server/resources/*.json",
-    "lint-utils": "jsonlint-cli server/utils/*.json",
     "prelint-js": "npm run create-rev",
     "lint-js": "eslint --ext=.js,.jsx gulpfile.js server/ common/ config/ client/",
-    "lint-json": "npm run lint-server && npm run lint-challenges && npm run lint-resources && npm run lint-utils",
+    "lint-json": "eslint --ext=.json seed/challenges/ server/",
```

Comment:
So i've come up with three solutions:
1. Eslint is ran twice, but condense into one long command.
`"lint": "eslint --ext=.js,.jsx gulpfile.js server/ common/ config/ client/ && eslint --ext=.json seed/challenges/ server/", `
2. Eslint is only ran once; however, this lints additional .js/.json files 
`"lint": "eslint --ext=.js,.jsx,.json gulpfile.js server/ common/ config/ client/ seed/challenges/ ",`
3. Appears to do the same as command 2
`"lint": "eslint --ext=.js,.jsx gulpfile.js server/ common/ config/ client/ --ext=.json seed/challenges/ server/", `

For commands 2 and 3 the following lint errors are produced:
```
/Users/arrowoode/Documents/FreeCodeCamp/freeCodeCamp/common/app/ns.json
  1:6  error  Missing semicolon  bad-json

/Users/arrowoode/Documents/FreeCodeCamp/freeCodeCamp/common/app/routes/challenges/ns.json
  1:13  error  Missing semicolon  bad-json

/Users/arrowoode/Documents/FreeCodeCamp/freeCodeCamp/common/app/routes/challenges/views/classic/ns.json
  1:10  error  Missing semicolon  bad-json

/Users/arrowoode/Documents/FreeCodeCamp/freeCodeCamp/common/app/routes/challenges/views/step/ns.json
  1:7  error  Missing semicolon  bad-json

✖ 4 problems (4 errors, 0 warnings)
```
If you'd like to use command 2, but ignore the files causing issues we can add `common/.json` to the .eslintignore file